Makefile

```diff
@@ -53,7 +53,7 @@ all: dev server benchkv
 parser:
 	@echo "remove this command later, when our CI script doesn't call it"
 
-dev: checklist check test 
+dev: checklist test check 
```

Comment:
why "check" is moved after "test"?

Reply:
IMO, I approve of `check` before `test`.
But to avoid something wrong like `checksum` before to block `make dev`, someone thinks the `check` after `test` is better.
In any case, it is innocuous.